server/models.py

```diff
@@ -260,6 +260,7 @@ class Status(models.TextChoices):
         models.PositiveIntegerField()
     )  # For Cross Pool and Brackets to have round number
 
+    name = models.CharField(max_length=10, null=True, blank=True)
```

Comment:
I think we can increase max_length here to accomodate "Quarter Finals" and all if we want to add like that

Reply:
Oh right, forgot to edit size of this field.